pages/suggestions/index.tsx

```diff
@@ -1,17 +1,47 @@
-import { FC } from "react";
+import { FC, useEffect, useState } from "react";
 import { PlusIcon } from "@heroicons/react/24/solid";
+import SuggestionCard from "@/components/cards/SuggestionCard";
+import { Instruments } from "@/constants/Instruments";
+import { getSuggestions } from "@/services/suggestions.service";
+import { ISuggestion } from "@/interfaces/Suggestion";
 
 const Suggestions: FC = () => {
+    const [suggestions, setSuggestions] = useState<ISuggestion[]>([])
+
+    const fetchData = async () => {
+        setSuggestions(await getSuggestions())
+    }
+
+    useEffect(() => {
+        (async () => fetchData())()
```

Comment:
You could just inline this 
`getSuggestions().then(suggestions => setSuggestions(suggestions))`